### ISSUE TITLE
Distribute on OS X homebrew

### ISSUE THREAD

**Issue by ghost** [edi9999/path-extractor#4]

**Comment by tomfitzhenry** [comment]
This seems to work, but Homebrew are unlikely to be happy accepting a version of a commit ID.

``` ruby
require "language/go"

class PathExtractor < Formula
  desc "Path extractor"
  homepage "https://github.com/edi9999/path-extractor"
  url "https://github.com/edi9999/path-extractor/tarball/54f890f142995fbc2ed737b8fff1635a8226ccc3"
  version "54f890f142995fbc2ed737b8fff1635a8226ccc3"
  sha256 "b3a92c07b1a96237e2069159fd3c64a4ce460f3649b21b5d5cab91e18a74616a"

  head do
    url "https://github.com/edi9999/path-extractor.git"
  end

  depends_on "go" => :build

  def install
    ENV["GOBIN"] = bin
    ENV["GOPATH"] = buildpath
    ENV["GOHOME"] = buildpath

    mkdir_p buildpath/"src/github.com/edi9999/"
    ln_sf buildpath, buildpath/"src/github.com/edi9999/path-extractor"
    Language::Go.stage_deps resources, buildpath/"src"

    system "go", "build", "-o", bin/"path-extractor", "path-extractor/pe.go"

  end

  test do
    assert True
  end

end
```

**Comment by edi9999** [comment]
If I tag the latest commit as v0.1.0 , will that be better ?

**Comment by tomfitzhenry** [comment]
Yes, Homebrew don't like unreleased software, it seems.

**Comment by edi9999** [comment]
I just tagged `v0.1.0`

**Comment by edi9999** [comment]
Done I think. I happen to have released a new version 0.2, don't know if you wish to open a new pull request
